### PR TITLE
Added checksum deprecation on record as for the native Kafka client

### PR DESCRIPTION
**Pull request by ppatierno**

### CODE REVIEW

File: src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.json.JsonArray;
+import java.util.Objects;
```
Comment (matzew) [inline src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java]: import by accident ?

Comment (vietj) [reply src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java]: this is generated code, so it has been added by the generator

Comment (ppatierno) [reply src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java]: yes thanks @vietj ;)
Do you think this PR is ok?

Comment (vietj) [reply src/main/generated/io/vertx/kafka/client/common/PartitionInfoConverter.java]: yes, this code will actually be regenerated without it at some point as we removed the source of it :-)

**Comment by vietj** [comment]
I think what is missing is the deprecation reason, i.e the original reason in Kakfa client should be in the doc as well.

**Comment by ppatierno** [comment]
do you mean adding the reason to the Javadoc of this Vert.x client right ?

**Comment by vietj** [comment]
yes, I think the original API that is deprecated must explain it, right ?

**Comment by ppatierno** [comment]
Yes yes, I'll just copy that explanation :-)

**Comment by ppatierno** [comment]
@vietj done!

**Pull request merged**